e2e-tests/pages/shared/autoCompleteField.js

```diff
@@ -0,0 +1,17 @@
+import fieldSelector from './field';
+
+/**
+ * autoCompleteField
+ * "Autocomplete" cypress selector.
+ * This field is used for any autocomplete fields, e.g country and currency fields.
+ * @param {String} fieldId: Field ID
+ * @returns
```

Comment:
* @returns {Object} autocompleted `input`, `results` and `noResults` Cypress selectors?
* Do we require `noResults` here given it is not being constructed with `fieldId`?